src/request/create/IdenticonSelector.js

```diff
@@ -134,7 +134,11 @@ class IdenticonSelector extends Nimiq.Observable {
      * @private
      */
     _onSelectionConfirmed() {
-        if (!this._selectedAddress) throw new Error('Invalid state');
+        if (!this._selectedAddress) { // something went wrong
+            this._clearSelection(); // clear current selection
+            this.generateIdenticons(); // and gerate new identicons
+            return;
+        }
```

Comment:
Add a `TODO` comment that this should result in a visible error message or hint to the user in the future.